de.dashup.shared/src/main/java/de/dashup/shared/Widget.java

```diff
@@ -2,12 +2,15 @@
 
 import com.google.gson.annotations.SerializedName;
 
-public class Widget extends DatabaseWidget implements Comparable<Widget>{
+import java.util.HashSet;
+import java.util.Set;
+
+public class Widget extends DatabaseWidget implements Comparable<Widget> {
 
     public enum Size {
         SMALL("small", "m2 s6"),
         MEDIUM("medium", "m4 s12"),
-        LARGE("large", "m5 s12");
+        LARGE("large", "m6 s12");
```

Comment:
I once changed it to m5 to have some space between the widgets in the central dashboard.

Reply:
I know, and I fixed this issue 😉